docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+npm run migrate
+exec npm run start
```

Comment:
I imagine the use of `exec` here is serves an important purpose but figured it couldn't hurt to ask!

Reply:
From what I have read it helps with signal handling in docker. I guess it drops the shell and lets npm be pid 1 within the container. So then whatever signals docker sends on `stop`, `kill`, etc., will go not to the parent bash process but to the intended `npm run start` process. Something like that. I haven't experimented with it to know more deeply but this is probably close.

Reply:
I had hoped I could ctrl-c to send sigint to the process when running with plain old `docker run ... ` but that doesn't appear to work yet either way. So maybe there is more improvement possible here. It seems to be OK under compose but I also have not looked very closely.